### PR TITLE
make the controller manager create and use a valid cobra command

### PR DESCRIPTION
**Pull request by deads2k**
The controller manager was returning an invalid cobra command that couldn't be executed and then side-stepped it by adding flags separately.  This brings it into line.

### CODE REVIEW

**Comment by deads2k** [comment]
/assign sttts

@kubernetes/sig-api-machinery-bugs

File: cmd/kube-controller-manager/app/controllermanager.go [hunk 1]
```diff
@@ -93,6 +93,7 @@ current state towards the desired state. Examples of controllers that ship with
 Kubernetes today are the replication controller, endpoints controller, namespace
 controller, and serviceaccounts controller.`,
 		Run: func(cmd *cobra.Command, args []string) {
+			Run(s)
```
Comment (ncdc) [inline cmd/kube-controller-manager/app/controllermanager.go]: You need to move `verflag.PrintAndExitIfRequested()` here

Comment (deads2k) [reply cmd/kube-controller-manager/app/controllermanager.go]: > You need to move verflag.PrintAndExitIfRequested() here as the first line in the func

done

**Pull request merged**